include/crm_internal.h

```diff
@@ -162,10 +162,6 @@ crm_clear_bit(const char *function, int line, const char *target, long long word
 {
     long long rc = (word & ~bit);
 
-    /* if(bit == 0x00002) { */
-    /*     crm_err("Bit 0x%.8llx for %s cleared by %s:%d", bit, target, function, line); */
-    /* } */
-
```

Comment:
It was left there on purpose, you'll want it when you next try and figure out why some action is being marked unrunnable/optional/etc.